quattor/types/aquilon/hardware.pan

```diff
@@ -0,0 +1,41 @@
+# This template extends the base schema with Aquilon-provided resources
+
+declaration template quattor/types/aquilon/hardware;
+
+include 'pan/types';
+include 'quattor/functions/hardware';
+include 'quattor/types/annotation';
+include 'quattor/types/sensors';
+include 'quattor/physdevices';
+
+@documentation{
+    Rack definition
+}
+type structure_rack = {
+    "column" : string
+    "name"   : string
+    "room"   ? string
+    "row"    : string
+};
+
+@documentation{
+    system location definition
+}
+type structure_sysloc = {
```

Comment:
In our version all fields of this structure are optional. I think it might be best to stay in sync.

Reply:
Fields changed to mandatory are those who are mandatory at the broker-level... See my [answer](https://github.com/quattor/template-library-core/pull/177#issuecomment-395180163) to your [comment](https://github.com/quattor/template-library-core/pull/177#pullrequestreview-126480852)

Reply:
@jouvin changing these to mandatory has broken builds with our (admittedly ancient) broker :disappointed:.